components/ProductMatrix.vue

```diff
@@ -1,16 +1,20 @@
 <template>
   <div>
-    <div v-if="error">
-      {{ error }}
-    </div>
-    <div class="product-matrix">
-      <div v-for="card in cards" :key="card.sortKey">
-        <component :is="components[card.type]" :card="card" />
+    <div ref="matrix" class="product-matrix">
```

Comment:
```suggestion
    <div class="product-matrix">
```

Reply:
I'm using this value to calculate columns

Reply:
Would it be an idea to name the reference "js-calculate-total-width" or something?

Reply:
We are referencing the "product matrix" to get the width from. I think having it named `matrix` is justified

Reply:
Then the name should be `product-matrix` to use the same name everywhere? Or change the class name --